frontend/build.rs

```diff
@@ -48,15 +64,55 @@ pub fn build_types() {
 pub fn build_frontend() {
     // Only install frontend dependencies when building release
     // #[cfg(not(debug_assertions))]
-    shell("npm install --frozen-lockfile");
+    shell("yarn");
 
     // Only build frontend when building a release
     // #[cfg(not(debug_assertions))]
-    shell("npm build");
+    shell("yarn build");
+}
+
+pub fn move_frontend() {
+    let mut target = PathBuf::from(std::env::var("OUT_DIR").unwrap()); // OUT_DIR == <project_dir>/target/<target_profile>/build/bob_management-<HASH>/out
+    let mut project_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    target.pop();
+    target.pop();
+    target.pop();
+    target.push(FRONTEND_DIR);
+
+    let mut file = std::fs::OpenOptions::new()
+        .append(true)
+        .open("build.log")
+        .expect("Couldn't open log file...");
+    file.write(format!("PROJECT DIR: {project_dir:?}\n").as_bytes())
+        .expect("Couldn't write to build log");
+
+    project_dir.push(FRONTEND_DIR);
```

Comment:
Place this line after `let project_dir = ...`. In this way it will be simpler to read the code